### PR TITLE
Create Rule “do-you-have-a-test-rule/rule”

### PR DESCRIPTION
**Pull request by LuciaSirtori**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by LuciaSirtori** [comment]
Hi @tiagov8 can you please check this Test rule for the SugarLEarning time for me?

**Comment by tiagov8** [comment]
Hi @LuciaSirtori 

Test pass! ✅

The only missing point is the URL which should have been shortened: https://ssw.com.au/rules/create-friendly-short-urls/

File: rules/do-you-have-a-test-rule/rule.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+   * Task board: XXX
+
+   &lt;This email was sent as per [Do you do Daily Scrums?](/methodology-daily-scrums)&gt;
+
+   :::
+   :::
+   ::: good
+   Figure: Schedule a recurring Daily Scrum meeting in Outlook using this template
+   :::
+4. In the afternoon, repeat step # 1 and make sure everyone follows the instructions n this video:
```
Comment (tiagov8) [inline rules/do-you-have-a-test-rule/rule.md:44]: @LuciaSirtori 

Noticed a small typo in this line

File: rules/do-you-have-a-test-rule/rule.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+::: greybox
+
+
+
+**Additional tips for the office:**
+
+* Always maintain a great attitude
+* Offer your help to your colleagues if you are available
+* Maintain a tidy and organised workspace
```
Comment (tiagov8) [inline rules/do-you-have-a-test-rule/rule.md:64]: @LuciaSirtori 

You should close the greybox with another `:::` at the end

**Pull request closed without merging**